presentation/src/main/res/values/colors.xml

```diff
@@ -3,4 +3,6 @@
     <color name="colorPrimary">#008577</color>
     <color name="colorPrimaryDark">#00574B</color>
     <color name="colorAccent">#D81B60</color>
+
+    <color name="redMarvel">#ED1C24</color>
```

Comment:
You should be using the primary color, check Santiago's PR, I think we defined those colors already.

Reply:
el que utiliza @SalosV me parece mas apagado, creo que se ve mejor con uno mas vivo

Reply:
We can have this discussion later, for now let's sync up and use the same color.